### ISSUE TITLE
[Logging v2] - Include the connected Outputs in ClusterFlow and Flow list views

### ISSUE THREAD

**Issue by ebauman** [rancher/dashboard#1569]
Rancher version: 2.5.0-rc4

Adding this information will help the user mentally (and visually) connect which outputs are receiving logs from a flow.

<img width="1171" alt="Screen Shot 2020-10-01 at 4 53 38 PM" src="https://user-images.githubusercontent.com/5565738/94867461-e7311300-0406-11eb-98b7-cd4ad978bfb1.png">

**Comment by codyrancher** [comment]
@ebauman are you okay with what we have on the overview page?
![image](https://user-images.githubusercontent.com/55104481/94874131-08453400-0406-11eb-8fc6-4175b9575e84.png)

**Comment by ebauman** [comment]
Yeah, those are fine I think.

**Comment by izaac** [comment]
Rancher version: `master-head (10/23/2020)` _a3a4c87_

ClusterFlow

I validated the Cluster Outputs and Configured Providers are shown properly in the list view.
Clicking on the names of the Cluster Outputs redirects the user to the proper resource.

Flow

I validated the Cluster Outputs, Outputs and Configured Providers are shown properly in the list view.
Clicking on the names of the Cluster Outputs redirects the user to the proper resource.
